### PR TITLE
Update en-us - i18n

### PR DESCRIPTION
**Pull request by Ella1117**

### CODE REVIEW

File: src/i18n/ui.ts [hunk 1]
```diff
@@ -52,48 +52,48 @@ export const ui = {
...
-    'footer.nav.company.concat': '联系我们',
-    'footer.nav.resource.title': '相关资源',
-    'footer.nav.resource.docs': '文档',
+    'tech.title': 'Technology spotlight',
+    'tech.t1.title': 'Dependency injection based plugin architecture',
+    'tech.t1.desc': 'Functions are divided into multiple plugins. Developers can use plugins freely and customize plugins to meet personalized needs.',
+    'tech.t2.title': 'Unified rendering engine',
+    'tech.t2.desc': 'TBD  ',
+    'tech.t3.title': 'Highly extensible command system',
+    'tech.t3.desc': 'All actions on Univer are executed through the command system. Based on it, univer provides advanced features like collaborative editing and cursor, live share, uniscript.',
```
Comment (Ella1117) [inline src/i18n/ui.ts:77]: All user actions on ....... through the command system, which makes it easy to implement advanced features such as collaborative editing + cursor tracking, live sharing and scripting.

File: src/i18n/ui.ts [hunk 2]
```diff
@@ -52,48 +52,48 @@ export const ui = {
...
+    'tech.t4.title': 'Self-developed formula engine',
```
Comment (Ella1117) [inline src/i18n/ui.ts:78]: Original formula engine

File: src/i18n/ui.ts [hunk 3]
```diff
@@ -52,48 +52,48 @@ export const ui = {
...
+    'tech.t4.desc': 'Univer provides features like inverted indexing, lambda expressions. Worker computation is implemented on the RPC mechanism, and server-side computation is on the way.',
```
Comment (Ella1117) [inline src/i18n/ui.ts:79]: Supports inverted indexing, lambda expressions, and more. Web worker compute is done behind RPC requests and allows transition into server side compute in the future.

File: src/i18n/ui.ts [hunk 4]
```diff
@@ -52,48 +52,48 @@ export const ui = {
...
-    'footer.button': '立即体验',
-    'footer.nav.company.title': '公司',
-    'footer.nav.company.concat': '联系我们',
-    'footer.nav.resource.title': '相关资源',
-    'footer.nav.resource.docs': '文档',
+    'tech.title': 'Technology spotlight',
+    'tech.t1.title': 'Dependency injection based plugin architecture',
+    'tech.t1.desc': 'Functions are divided into multiple plugins. Developers can use plugins freely and customize plugins to meet personalized needs.',
+    'tech.t2.title': 'Unified rendering engine',
+    'tech.t2.desc': 'TBD  ',
```
Comment (Ella1117) [inline src/i18n/ui.ts:75]: The high-performance rendering engine, based on a single canvas, allows customized layouts with rich text, sheets, graphics to be interleaved and manipulated freely.

File: src/i18n/ui.ts [hunk 5]
```diff
@@ -52,48 +52,48 @@ export const ui = {
...
-    'footer.nav.resource.title': '相关资源',
-    'footer.nav.resource.docs': '文档',
+    'hero.title': 'The next-generation open source office suite',
+    'hero.description': 'Break application silos and connect business data to a single platform.',
+    'hero.button': 'Get started',
+    'features.title': 'Product highlights',
+    'features.f1.title': 'Data connector, integrate business data',
+    'features.f1.desc': ['Import external data across platforms, automate periodic tasks with data sync.'],
+    'features.f2.title': 'Script editor, record operation commands',
+    'features.f2.desc': ['Create scripts for task automation, AI-generated script would be involved later.'],
```
Comment (AMOOOMA) [inline src/i18n/ui.ts:66]: would be involved ->  could be integrated in the future.

File: src/i18n/ui.ts [hunk 6]
```diff
@@ -52,48 +52,48 @@ export const ui = {
...
+    'features.f3.title': 'Boundaryless mode, merge Docs,Slides,Sheets in a file',
```
Comment (AMOOOMA) [inline src/i18n/ui.ts:67]: Docs, Slides, Sheets

(need spaces)

File: src/i18n/ui.ts [hunk 7]
```diff
@@ -52,48 +52,48 @@ export const ui = {
...
+    'features.f3.desc': ['Use unified functions in a file, like formulas and script editor.'],
+    'features.f4.title': 'collaboration, make your team work closer ',
+    'features.f4.desc': ['Edit simultaneous in your team, and comment on anything you want to discuss.'],
```
Comment (AMOOOMA) [inline src/i18n/ui.ts:70]: simultaneously

File: src/i18n/ui.ts [hunk 8]
```diff
@@ -52,48 +52,48 @@ export const ui = {
...
+    'hero.title': 'The next-generation open source office suite',
+    'hero.description': 'Break application silos and connect business data to a single platform.',
+    'hero.button': 'Get started',
+    'features.title': 'Product highlights',
+    'features.f1.title': 'Data connector, integrate business data',
+    'features.f1.desc': ['Import external data across platforms, automate periodic tasks with data sync.'],
+    'features.f2.title': 'Script editor, record operation commands',
+    'features.f2.desc': ['Create scripts for task automation, AI-generated script would be involved later.'],
+    'features.f3.title': 'Boundaryless mode, merge Docs,Slides,Sheets in a file',
+    'features.f3.desc': ['Use unified functions in a file, like formulas and script editor.'],
```
Comment (AMOOOMA) [inline src/i18n/ui.ts:68]: Use unified functions in any file or forms, embedding formulas and scripts across docs, sheets, and slides.

**Pull request merged**